### ISSUE TITLE
read_json() fails with file path argument

### ISSUE THREAD

**Issue by adamgreg** [pola-rs/polars#2748]
#### What language are you using?

Python

#### Which feature gates did you use?

This can be ignored by Python & JS users.

#### What version of polars are you using?

0.13.5

#### What operating system are you using polars on?

Windows 10

#### What language version are you using
Python 3.8

#### Describe your bug.

The docstring for `polars.read_json()` says that the argument can be a string "Path to a file". Passing a string file path fails, as the argument is actually interpreted as a JSON string, which is undocumented.

#### What are the steps to reproduce the behavior?

```python
import polars as pl

pl.read_json('path/to/file.json')  # This fails
pl.read_json('{"columns": [{"name": "a", "datatype": "Int64", "values": [1,2,3]}]}')  # This works, but is undocumented
```

#### What is the actual behavior?

```python
RuntimeError: Other("Error(\"expected value\", line: 1, column: 1)")
```

#### What is the expected behavior?

It should be able to read the JSON from a file given the path as a string.

The fact that Polars can read JSON directly from a string argument should be mentioned in the docstring.

It would also be nice to add support for `pathlib.Path` arguments.

**Comment by adamgreg** [comment]
This seems pretty simple, so I'd like to have a crack at fixing it myself in a PR if that's ok.

**Comment by ritchie46** [comment]
> This seems pretty simple, so I'd like to have a crack at fixing it myself in a PR if that's ok.

Please :)

**Comment by ritchie46** [comment]
Fixed by #2761

**Comment by adamgreg** [comment]
I was too slow! Thanks for such a quick fix
